### PR TITLE
Feature/cesium support

### PR DESCRIPTION
**Pull request by bertt**

### CODE REVIEW

File: src/BoundingBox3D.cs [hunk 1]
```diff
@@ -94,5 +91,27 @@ public Vector3 GetCenter()
             var z = (ZMax + ZMin) / 2;
             return new Vector3((float)x, (float)y, (float)z);
         }
+
+        public double[] GetBoundingvolumeBox() {
```
Comment (bertt) [inline src/BoundingBox3D.cs:95]: box we don't use anymore? only region?

File: src/Program.cs [hunk 2]
```diff
@@ -8,17 +8,21 @@
...
 using System.Linq;
+using System.Threading;
 
 namespace i3dm.export
 {
     class Program
     {
         static void Main(string[] args)
         {
+            Thread.CurrentThread.CurrentCulture = System.Globalization.CultureInfo.InvariantCulture;
```
Comment (bertt) [inline src/Program.cs:19]: can be removed? use InvariantCulture when needed

File: src/ECEF/ECEFMath.cs [hunk 3]
```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Numerics;
+
+namespace i3dm.export.ECEF
+{
+    public static class ECEFMath 
+    {
+        public static bool EqualsEpsilon(float left, float right, float relativeEpsilon, float absoluteEpsilon = 0.0f)
```
Comment (bertt) [inline src/ECEF/ECEFMath.cs:8]: need unit tests for these calculations

File: src/BoundingBox3D.cs (hunk 1 above)
Comment (tebben) [reply src/BoundingBox3D.cs:95]: box is still used for mapbox export, mapbox3dtiles currently does not support region

File: README.md [hunk 4]
```diff
@@ -78,6 +78,8 @@ Tool parameters:
 --use_external_model: (optional - default false) Use external model instead of embedded model
 
 --use_scale_non_uniform: (optional - default false) Use column scale_non_uniform for scaling instances
+
+--cesium: (optional - default false) Export tileset that is compatible with Cesium
```
[the remark this answers is not in the document]
Comment (tebben) [reply README.md]: I agree, updated in a new commit

File: src/Program.cs (hunk 2 above)
Comment (tebben) [reply src/Program.cs:19]: I'm not sure what the best option is. The coordinates in queries https://github.com/Geodan/i3dm.export/blob/feature/cesium-support/src/InstancesRepository.cs#L15 and https://github.com/Geodan/i3dm.export/blob/feature/cesium-support/src/InstancesRepository.cs#L37 result in wrong numbers on my machine, this is fixed by setting the thread to InvariantCulture.

I could add a function ToInvariantCulture and run all the coordinates trough this?

**Pull request merged**